README.md

```diff
@@ -88,7 +88,6 @@ Display the [3scale](http://3scale.net) CLI help with the following command
   Commands:
 
     import [options]     Import an API from it's API definition (swagger or RAML) into 3scale
-    lambda [options]     Generate the Lambda function to integrate with 3scale
     config               Configure the 3scale cli
     appplan [options]    Adds an application plan to a service
```

Comment:
Actually on the rest it does not matter as it's not "user-facing"
I also changed it in the README

Also it cant be change on the `-a, --appplan`  parameter because I dont think we can have parameters with `-`.